### ISSUE TITLE
Accessibility issue in UVPostIdeaViewController.m

### ISSUE THREAD

**Issue by ericanylee** [uservoice/uservoice-ios-sdk#454]
When you have VoiceOver on, click on Post an idea, then click on the textview. The VoiceOver should be reading "text field, Description, optional" or something alone the lines. Instead it reads "Empty line" which may be confusing to the users. 

this may be fixed by inserting a line of code on UVPostIdeaViewController.m  at line 54:

    _fieldsView.textView.accessibilityTraits = UIAccessibilityTraitStaticText;
